test/unit_tests/braket/default_simulator/test_state_vector_simulator.py

```diff
@@ -666,10 +666,17 @@ def test_simulator_identity(caplog):
     assert not caplog.text
 
 
-@pytest.mark.xfail(raises=TypeError)
 def test_simulator_instructions_not_supported(circuit_noise):
     simulator = StateVectorSimulator()
-    simulator.run(circuit_noise, qubit_count=2, shots=0)
+    no_noise = re.escape(
+        "Noise instructions are not supported by the state vector simulator (by default). "
+        'You need to use the density matrix simulator: LocalSimulator("braket_dm").'
+    )
+    with pytest.raises(TypeError, match=no_noise):
```

Comment:
You said in the description it wasn't testing what it was meant to -- was it that a type error would be raised if qubit count was supplied for non-Jaqcd?

Reply:
That is the type error that was getting raised, but this test is supposed to test that a type error gets raised by noise instructions in an otherwise valid circuit (for both oq and jaqcd). Openqasm programs don't take a qubit count argument, since that isn't known until interpretation of the program source